content/release/10/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Jakarta EE 10"
+date: 2022-04-26T08:10:38-04:00
+description: "The Jakarta EE Working Group, hosted by the Eclipse Foundation, one of the world’s largest open source software foundations, today announced the release of the Jakarta EE 10 Platform and Web Profile specifications and related TCKs."
+hide_sidebar: true
+seo_title: "Jakarta EE 10 Downloads | Java EE 10 Compatible Products & Specifications"
+headline: "<img class='header-release-10-logo' src='images/jakarta10-logo.svg' alt='Jakarta EE 10'>"
+logo: "images/jakarta/jakarta_ee_logo_schooner_color_horizontal_default.png"
+links: [[href: "/compatibility/", text: "Download Compatible Products"], [href: "/specifications/", text: "Specifications"]]
+hide_page_title: true
+header_wrapper_class: "header-release-10"
+container: "container-fluid"
+footer_class: "footer-release-9"
+type: "release/10"
+hide_call_for_action: true
+press_release_link: "/news/jakarta-ee-9-1-released/"
```

Comment:
Here should be press_release_link for Jakarta EE 10. (probably should be added when published, or provided link that will be used with anouncements)